src/main/docker/Dockerfile

```diff
@@ -83,11 +83,7 @@ RUN chgrp 0 /usr/share/opensearch/opensearch-docker-entrypoint.sh && \
 #
 # The actual image definition
 #
-FROM @projectDockerRegistry@/cafapi/opensuse-jre17
-
-RUN zypper -n refresh && \
```

Comment:
This updated the permissions on the nssdb directory and caused failure to initialize NSS database in classification worker.